halyard-config/src/test/groovy/com/netflix/spinnaker/halyard/model/v1/UpdateableSpec.groovy

```diff
@@ -168,3 +180,23 @@ public class AccountValidator extends Validator<MyAccount> {
     return false
   }
 }
+
+public class VersionValidator extends Validator<MyAccount> {
+  public VersionValidator(Halconfig halconfig, MyAccount subject) {
+    super(halconfig, subject)
+  }
+
+  @Override
+  Stream<String> validate() {
+    if (context.halyardVersion == null) {
+      return Stream.of(UpdateableSpec.ERROR_MESSAGE)
+    } else {
+      return null
+    }
+  }
+
+  @Override
+  boolean skip() {
```

Comment:
Another reason was that (at least for me) I think of "enable" as something determined by the user, external to what they are enabling/disabling. In this case, the validator itself determines if it should skip a validation, so it feels weird to have it "enable/disable" itself.